### DISCUSSION TITLE
Add an option to hide URL in the Default Page (blocked.html) to prevent an obvious workaround

### DISCUSSION THREAD

**Discussion by Rodvi** [proginosko/LeechBlockNG#452] [Ideas]
I have a block set of keywords. When any of these keywords is matched in some website then the so-called Default Page appears. The big problem for me is that this page contains the URL of the website. It has the following obvious workaround, which I unfortunately use very often. It is enough to right-click on the URL, then the context menu will appear with an option to save a page (as .htm or .html file). Saved page will contain all blocked content. So I really need an option to hide URL in the Default Page (blocked.html).
 
Note that I can't use the Blank Page (instead the Default Page) because it doesn't contain matched keyword. I need to see this keyword to understand why the website was blocked. And also because I want to see the name of the block set.

Finally, I think it will also help to prevent the save-page workaround for block sets that contain URLs only – in the case of using the Default Page (if a person wants to see the name of the block set, they can't use the Blank Page).

**Comment by proginosko** [comment]
This is easy to implement. Look for it in the next release!

**Comment by proginosko** [comment]
Added in version 1.6.5.

**Comment by Rodvi** [comment]
I just tested "Disable hyperlink to blocked page on blocking/delaying page" and this works great! Thanks a lot!
